ui/src/tasks/actions/v2/index.ts

```diff
@@ -357,7 +362,8 @@ export const saveNewScript = (
     dispatch(populateTasks())
   } catch (e) {
     console.error(e)
-    dispatch(notify(taskNotCreated(e.message)))
+    const message = _.get(e, 'response.data.error.message', '')
```

Comment:
could you use getErrorMessage here?

Reply:
Actually, don't need it here because the api comes back with response.data.error.message.